noir/noir-repo/tooling/nargo_cli/src/cli/test_cmd.rs

```diff
@@ -137,7 +137,7 @@ fn run_tests<S: BlackBoxFunctionSolver<FieldElement> + Default>(
     println!("[{}] Running {count_all} test function{plural}", package.name);
 
     let test_report: Vec<(String, TestStatus)> = test_functions
-        .into_par_iter()
```

Comment:
This can be reverted after https://github.com/noir-lang/noir/pull/5205

Reply:
Awesome, thanks Tom! Are there any blockers to synching noir soon-ish?

Reply:
This has been synced now.